servers/quarkus-server/src/test/java/org/projectnessie/server/AbstractTestBasicOperations.java

```diff
@@ -47,10 +48,8 @@ void closeClient() {
   }
 
   void getCatalog(String branch) throws BaseNessieClientServerException {
-    api =
-        HttpClientBuilder.builder()
-            .withUri("http://localhost:19121/api/v1")
-            .build(NessieApiV1.class);
+    String quarkusUrl = Objects.requireNonNull(System.getProperty("test.url"));
```

Comment:
I didn't get how this property name (`"test.url"`) is chosen.

Because in application.properties, `quarkus.http.test-port` is the one we are changing to dynamic.

I thought some code is translating it. But I couldn't find it.

Reply:
I saw this comment now https://github.com/projectnessie/nessie/pull/5352#discussion_r992110450

But still not sure how it is translated.

Reply:
i printed all system properties when running these tests in intellij and saw that `test.url` seemed to be the only one injected automatically

i think it is set here by quarkus:
https://github.com/quarkusio/quarkus/blob/c7c6eab48e71ea782f98f0884ad624981cb1b802/test-framework/common/src/main/java/io/quarkus/test/common/LauncherUtil.java#L188-L196

according to my understanding of https://quarkus.io/guides/getting-started-testing#controlling-the-test-port
you set the `quarkus.http.test-port` in the quarkus application properties but it does not translate directly to a system property?
all the examples from that page use various quarkus annotations/injected resources etc... i assume internally they are reading stuff directly from the quarkus config (which is not available as system property?)

Reply:
when I was debugging for Sagar's PR, I remember seeing dynamic port value in `System.getProperty("quarkus.http.test-port")`. Hence, it triggered my doubts.

Reply:
i re-checked and yeah it is there:
```
test.url -> http://localhost:38269
quarkus.http.test-port -> 38269
%test.quarkus.http.port -> 38269
quarkus.http.port -> 38269
```
not sure how i missed it originally... however `test.url` might still be the right thing to use?

Reply:
i dont think it matters too much but ok will use that instead (for consistency with the other classes i.e. delta + spark)